### PR TITLE
[15.0][FIX] l10n_de, l10n_fr: company view should inherit from 'account'

### PR DESCRIPTION
**Pull request by sebalix**
### Description of the issue/feature this PR addresses:

To find the `country_code` field, `l10n_de` and `l10n_fr` addons have to inherit from `account` company form.

### Current behavior before PR:

Upgrade of `l10n_de` and `l10n_fr` are failing.

### Desired behavior after PR is merged:

Smooth upgrade.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

File: addons/l10n_de/views/res_company_views.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <record id="res_company_form_l10n_de" model="ir.ui.view">
             <field name="name">res.company.form</field>
             <field name="model">res.company</field>
-            <field name="inherit_id" ref="base.view_company_form"/>
+            <field name="inherit_id" ref="account.view_company_form"/>
```
Comment (ajaniszewska-dev) [inline addons/l10n_de/views/res_company_views.xml:6]: @sebalix  can you please update also l10n_de_reports?

Comment (sebalix) [reply addons/l10n_de/views/res_company_views.xml:6]: This one is part of `odoo/enterprise`, I'll open the PR there yes.

**Comment by sebalix** [comment]
@vin-odoo we are performing a simple update of `base` on a database migrated to 15.0 by your Upgrade Service, I would say a simple `upgrade` of `base` should be enough to reproduce this.

**Comment by qdp-odoo** [comment]
@robodoo r+ rebase-ff

Thanks for the contribution, guys!

**Pull request closed without merging**